depends/packages/miniupnpc.mk

```diff
@@ -12,6 +12,10 @@ $(package)_build_opts_mingw32=-f Makefile.mingw
 $(package)_build_env+=CFLAGS="$($(package)_cflags) $($(package)_cppflags)" AR="$($(package)_ar)"
 endef
 
+# sed is used here to replace the host and version that will end up as part of
+# the miniupnpc User-Agent string, that will ultimately end up in bitcoind. i.e
```

Comment:
Which symbol or variable name contains this ua string? Not the p2p ua string of the `version` message, right?

Reply:
no, not the P2P UA string, it's an UA string sent to the UPnP server (so broadcasted to the network you'[re connected to, if you have UPnP enabled)